lib/auth/auth_with_roles_test.go

```diff
@@ -8063,11 +8063,32 @@ func TestCreateAccessRequest(t *testing.T) {
 	})
 	require.NoError(t, err)
 
-	requestRole, err := types.NewRole("requestRole", types.RoleSpecV6{})
```

Comment:
This role wasn't created before, because `searchRole` above had the same name (`"requestRole"`) and `srv.Auth().CreateRole(ctx, requestRole)` call (below) wasn't checked for error.